src/com/ableton/VirtualEnv.groovy

```diff
@@ -171,13 +161,7 @@ class VirtualEnv implements Serializable {
    * @param command Command to run.
    */
   void run(String command) {
-    script.sh(
-      label: "Run command in virtualenv: ${command}",
```

Comment:
Why are we losing the `label` in 7ebbf7202ef60be4de9ea2299ac34fb5545e7116?

Reply:
Fixed in 750094b.